targets/mgyamlfmt/targets.go

```diff
@@ -26,14 +25,31 @@ func FormatYaml() error {
 	})
 }
 
+// PreserveEmptyLines adds a temporary #comment on each empty line in the provided byte array.
+// CleanupPreserveEmptyLines can be used to clean up the temporary comments.
+func PreserveEmptyLines(src []byte) []byte {
```

Comment:
can be private?

Reply:
Its public so I can also use it when appending to dependabot from the init command

Reply:
Ahh!